.github/workflows/github.yml

```diff
@@ -72,22 +73,33 @@ jobs:
             conan_prebuilts: dependencies-ios
             conan_options: --options with_apple_system_libs=True
           - platform: msvc-x64
+            arch: x64
             os: windows-latest
             test: 0
-            pack: 1
+            pack: 0
```

Comment:
Might be better to keep packing, but instead switch to .zip format (should be supported by CPack).

Probably no need to upload such .zip's to vcmi.eu, but would be convenient to have in CI artifacts

Reply:
I have disabled it as currently it created old NSIS one. Have not removed NSIS yet. Keeping this for future cleanup PR.
Adding ZIP release could be easy step in CI with same file ignore filter as is in ISS script.